.circleci/config.yml

```diff
@@ -51,5 +52,10 @@ workflows:
           vcs-type: << pipeline.project.type >>
           pipeline-number: << pipeline.number >>
           filters:
+            branches:
```

Comment:
lets remove this and instead just get rid of the commands and jobs being tested in test-deploy.yml